recipes/uwebsockets/all/conanfile.py

```diff
@@ -79,6 +79,9 @@ def validate(self):
                 % (self.name, minimal_cpp_standard)
             )
 
+        if self.settings.compiler == "clang" and str(self.settings .compiler.libcxx) == "libstdc++":
```

Comment:
```suggestion
        if tools.Version(self.version) >= "20.14.0" self.settings.compiler == "clang" and str(self.settings .compiler.libcxx) == "libstdc++":
```

I dont see that error for 20.9.0